app/pathogen/arbovirus/visualizations/page.tsx

```diff
@@ -1,45 +1,16 @@
 "use client"
```

Comment:
Only note here is that it probably makes sense to merge the visualization and (visualization) directories. I understand why they would be seperate technically but they are never used without each other and hence lets centralize it - and avoid confusion going forward

Reply:
Let me see what we can do once there's less open PRs hahaha :)